src/Function/Bundles.agda

```diff
@@ -189,6 +189,51 @@ module _ (From : Setoid a ℓ₁) (To : Setoid b ℓ₂) where
       }
 
 
+  record BiEquivalence : Set (a ⊔ b ⊔ ℓ₁ ⊔ ℓ₂) where
+    field
+      f     : A → B
+      g₁    : B → A
+      g₂    : B → A
+      cong₁ : f Preserves _≈₁_ ⟶ _≈₂_
+      cong₂ : g₁ Preserves _≈₂_ ⟶ _≈₁_
+      cong₃ : g₂ Preserves _≈₂_ ⟶ _≈₁_
```

Comment:
Maybe there could be some more descriptive names for these fields, e.g. `cong-f`, `cong-g₁`, and `cong-g₂`?

Reply:
I agree, that notation is more descriptive and clear, maybe @MatthewDaggitt can say something about the impact of this change. In order to preserve consistency, `cong₁` and `cong₂` should be also elsewhere in the library replaced by `cong-f` and `cong-g` respectively.

Reply:
I believe we usually name things `f-cong` and use `Congruent`.
Cf. [`Algebra.Structures`](https://agda.github.io/agda-stdlib/Algebra.Structures.html)